### ISSUE TITLE
Update for Rollup v1

### ISSUE THREAD

**Issue by paulmelnikow** [saf33r/rollup-plugin-cleaner#5]
I'm using 0.2.0 and seeing this deprecation warning on Rollup 1.0.0.

```
(!) cleaner plugin: The ongenerate hook used by plugin cleaner is deprecated. The generateBundle hook should be used instead.
```

**Comment by paulmelnikow** [comment]
Fixed in #7.

**Comment by paulmelnikow** [comment]
Published at 1.0.0! Thanks for the access @saf33r! 🎉
